src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -49,6 +49,31 @@ public boolean connect(String database, String user, String password) {
         }
     }
 
+    public boolean insert(String tableName, DataSet dataset) {
+        String csvValues = csvValues(dataset);
+        String sqlQuery = String.format("INSERT INTO %s VALUES(%s)", tableName, csvValues);
+        try(Statement statement = connection.createStatement()){
+            statement.executeUpdate(sqlQuery);
+            return true;
+        } catch (SQLException e) {
+            return false;
+        }
+    }
+
+    private String csvValues(DataSet dataset) {
```

Comment:
CSV as I know is  'Comma-Separated Values'. https://ru.wikipedia.org/wiki/CSV
Maybe rename method and variabel as csvText or csvRow or something similar?

Reply:
Yup! You're right, I will rename thit method to csvEntry. Thanks!